source/js/main.js

```diff
@@ -61,6 +62,20 @@ function recipesPage() {
   renderRecipes(Object.keys(storage.recipeData), recipeSection);
 }
 
+/**
+ * This function would replace the main with the search page including the filter and the body
+ */
+function SearchPage() {
```

Comment:
```suggestion
function SearchPage(results) {
```

Reply:
I have added them. Do I start a new pull request?

Reply:
no need, if you commit them to this branch, this PR will automatically update